docs/configuration.rst

```diff
@@ -0,0 +1,19 @@
+Configuration
```

Comment:
There is another setting `DJANGO_CELERY_RESULTS_TASK_ID_MAX_LENGTH` which should probably be added here but I'd rather do this separately if you don't mind.